app/assets/javascripts/protocols_show.js.coffee

```diff
@@ -4,14 +4,11 @@ $ ->
 
     # Bootstrap Tab persistence
 
```

Comment:
If we don't set the BS tab 'active' class in the haml at all, then all we need to do is: https://gist.github.com/goleksiak/a4d964f18057bd6349f7 in protocol_show.coffee. The user won't see the wrong tab flash by...